shim/third-party/rust/fixups/serde_json/fixups.toml

```diff
@@ -1 +1,9 @@
 buildscript = []
+
+[platform_fixup.'cfg(any(target_arch = "aarch64", target_arch = "loongarch64", target_arch = "mips64", target_arch = "powerpc64", target_arch = "wasm32", target_arch = "x86_64"))']
+buildscript = []
+rustc_flags = ['--cfg=fast_arithmetic="64"']
+
+[platform_fixup.'cfg(not(any(target_arch = "aarch64", target_arch = "loongarch64", target_arch = "mips64", target_arch = "powerpc64", target_arch = "wasm32", target_arch = "x86_64")))']
+buildscript = []
+rustc_flags = ['--cfg=fast_arithmetic="32"']
```

Comment:
Please run the upstream build script, instead of transcribing the logic like this.

Example:

https://github.com/facebook/buck2/blob/9beff407c2a67de0b0f28dc900dbf50841d2f14a/shim/third-party/rust/fixups/num-traits/fixups.toml#L1-L2

Reply:
sgtm, I didn't realize thats what the syntax does